packages/prettier-config/index.js

```diff
@@ -3,6 +3,7 @@
 module.exports = {
   singleQuote: true,
   bracketSpacing: true,
+  printWidth: 100,
```

Comment:
Why not 120 as well?

Reply:
According to [prettier docs](https://prettier.io/docs/en/options.html#print-width):

> Prettier’s printWidth option does not work the same way. It is not the hard upper allowed line length limit. It is a way to say to Prettier roughly how long you’d like lines to be. Prettier will make both shorter and longer lines, but generally strive to meet the specified printWidth.

It is possible it will sometimes makes a bit longer lines. It does not server as a hard limit, but "recommended" max length. 

While editorconfig sets the max-length, where IDE will attempt to make the line break no matter what.

So I thought this could cause conflicts: with 120, prettier will make eg. 125 chars long line. And your IDE will then break it to 120 chars. And prettier check will then fail, and will try to return the line back to 125 chars.

But maybe I understand the prettier behavior wrongly 🤷

Reply:
> It is a way to say to Prettier roughly how long you’d like lines to be.

OK 🙈